requirements-minimal.txt

```diff
@@ -10,7 +10,6 @@ certifi
 choice >= 0.1
 cookiecutter >= 1.4.0
 croniter
-dataclasses >= 0.7; python_version < '3.7'
```

Comment:
we use py37 - no need to conditionally include this backport :p

Reply:
This wouldn't be true for package-based use cases; this file is used to populate the `install_requires` in `setup.py`, and there isn't a requirement that users of the `paasta-tools` package use py37 (e.g. `python_requires='>=3.7.0'`

Reply:
ugh, i forgot that we use paasta as a library in some places ;_; - lemme add this case explicitly in our setup.py since I don't want this to snowball into dropping support for a. python version that something might still be using internally (and that deps paasta)

Reply:
actually, it looks like there's only one usecase at yelp that uses py36 and has paasta as a dep so i'm gonna go ahead and add a python_requires to gently nudge them :p